components/eam/src/physics/crm/pam/pam_driver.cpp

```diff
@@ -150,12 +220,15 @@ extern "C" void pam_driver() {
     dycore.declare_current_profile_as_hydrostatic(coupler,/*use_gcm_data=*/true);
   #endif
 
+  #ifdef MMF_DISABLE_DENSITY_RECALL
+    do_density_save_recall = false;
+  #endif
   //------------------------------------------------------------------------------------------------
   //------------------------------------------------------------------------------------------------
   //------------------------------------------------------------------------------------------------
 
   // set number of CRM steps
-  int nstop = int(gcm_dt/crm_dt);
+  int nstop = int(gcm_dt/crm_dt_fixed);
```

Comment:
This has an implied `floor` operation. Is that what you intend if they aren't evenly divisible? Another way to handle non-evenly divisible could be:
```C++
int nstop = std::ceil(gcm_dt / crm_dt_fixed);
crm_dt_fixed = gcm_dt / nstop
```

Reply:
I'm actually not sure which is better or if it even matters.... so I'm just gonna leave the implied floor in there.